src/pkcs15init/pkcs15-oberthur-awp.c

```diff
@@ -1078,10 +1065,9 @@ awp_encode_cert_info(struct sc_pkcs15_card *p15card, struct sc_pkcs15_object *ob
 err:
 	ERR_print_errors_fp(stderr);
 	ERR_clear_error();
-	ERR_free_strings();
 	if (pubkey.exponent.data) free(pubkey.exponent.data);
 	if (pubkey.modulus.data) free(pubkey.modulus.data);
-	if (x) 		X509_free(x);
+	if (x)		X509_free(x);
```

Comment:
```suggestion
	if (x) X509_free(x);
```
i would suggest to keep a space here instead of the tab. Or just expand it to separate lines to keep it more readable when you touch this part already.

Reply:
OK

Reply:
I do not see this minor issue fixed in the final verison